cmd/homi/setup/cmd.go

```diff
@@ -365,7 +365,7 @@ func genServiceChainGenesis(nodeAddrs, testAddrs []common.Address) *blockchain.G
 	genesisJson.Config.Governance.Reward.StakingUpdateInterval = 86400
```

Comment:
These code (intervals, minimumStake is necessary? @laggu

Reply:
governance can be nil.
all configure in governance will be default values.
only consider that governance mode will be none and governingnode will be 0x0

Reply:
Fixed.